projects/ad9739a-fmc-ebz/src/ad9739a_fmc_ebz.c

```diff
@@ -169,6 +164,33 @@ int main(void)
 		return FAILURE;
 	}
 
+#ifdef DAC_DMA_EXAMPLE
+	extern const uint16_t sine_lut[128];
+	struct axi_dmac_init ad9739a_dmac_init_param = {
+		.name = "ad9739a_dmac",
+		.base = TX_DMA_BASEADDR,
+		.direction = DMA_MEM_TO_DEV,
+		.flags = DMA_CYCLIC
+	};
+	struct axi_dmac *ad9739a_dmac_desc;
+
+	ad9739a_channels[0].sel = AXI_DAC_DATA_SEL_DMA;
```

Comment:
I think we should use functions like _axi_dac_set_datasel_ to modify this parameters no directly the structures.

Reply:
I wouldn't let the channel attributes completely uninitialized.

Also, if you call directly axi_dac_set_datasel, and you want to change all channels, you will have to add raw negative value as parameter. 
`axi_dac_set_datasel(ad9739a_core, -1, AXI_DAC_DATA_SEL_DMA)`
For one channel:
`axi_dac_set_datasel(ad9739a_core, 0, AXI_DAC_DATA_SEL_DMA)`

I'm not a fan of this approach, since these values are not mapped accordingly yet.

For now `axi_dac_data_setup` wraps properly these kind of calls.